protoc.go

```diff
@@ -53,9 +55,16 @@ func (p *protocCmd) run() error {
 		log.Fatalln(err)
 	}
 
+	shcmd := "sh"
+	sharg := "-c"
+	if runtime.GOOS == "windows" {
```

Comment:
This will have odd behavior when running under a build targeted at windows. The issue is that this may be run on a linux host with GOOS set to windows. In that case, we'd still shell out to `sh` but the underlying build will target windows. What I am not sure about here is whether `runtime.GOOS` picks up the environment variable or is set in the binary at build time.

Placing this in a separate binary with build tags would prevent this from being an issue.

Let me know if you need direction on how to achieve that.